### PR TITLE
Minor PEP8/Cleaning changes

### PR DESCRIPTION
**Pull request by CBroz1**
- PEP8
- `TODO` -> issues
- Added YouTube link to README

### CODE REVIEW

**Comment by kabilar** [comment]
Please add YouTube link to README.

```
# Video Tutorial
[![DataJoint Element Miniscope](https://img.youtube.com/vi/nWUcPFZOSVw/0.jpg)](https://www.youtube.com/watch?v=nWUcPFZOSVw)
```

**Comment by CBroz1** [comment]
> Please add YouTube link to README

In the element and not workflow? My gut says this fits best under the `Interacting with the DataJoint workflow` section of the workflow README

**Comment by kabilar** [comment]
> In the element and not workflow? My gut says this fits best under the `Interacting with the DataJoint workflow` section of the workflow README

Good point.  How about both in order to maximize opportunities for users to see the link?

File: element_miniscope/miniscope.py [hunk 1]
```diff
@@ -233,19 +238,19 @@ def make(self, key):
...
             px_width = metadata['ROI']['width']
-            fps = int(metadata['frameRate'].replace('FPS',''))
+            fps = int(metadata['frameRate'].replace('FPS', ''))
             gain = metadata['gain']
-            spatial_downsample = 1 # Assumes no spatial downsampling
+            spatial_downsample = 1  # Assumes no spatial downsampling
             led_power = metadata['led0']
             time_stamps = np.array([list(map(int, time_stamps[i]))
-                                       for i in range(1,len(time_stamps))])
+                                    for i in range(1, len(time_stamps))])
```
Comment (dimitri-yatsenko) [inline element_miniscope/miniscope.py:253]: Does this really intend to skip `time_stamps[0]`?

Comment (kabilar) [reply element_miniscope/miniscope.py:253]: Thanks @dimitri-yatsenko.  This is implemented to skip the heading of the CSV file:
image

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!

**Pull request merged**